.devcontainer/library-scripts/go-debian.sh

```diff
@@ -16,6 +16,9 @@ USERNAME=${4:-"automatic"}
 UPDATE_RC=${5:-"true"}
 INSTALL_GO_TOOLS=${6:-"true"}
 
+# https://www.google.com/linuxrepositories/
```

Comment:
This script was updated from the vscode-dev-containers repository as an issue was occurring when building the dev-container locally

https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/go-debian.sh